### ISSUE TITLE
Distances by trip purpose seem off

### ISSUE THREAD

**Issue by moeckel** [msmobility/mito#140]
In MITO, trip lengths for several purposes seem to be off. Work trips seem to be too long, and in particular non-home based trips seem way too long. 
@MtthsLanger : Could you summarize from MiD the average trip length by trip purpose?
@joanna-ji : Once Matthias provided you new trip lengths by purpose, could you recalibrate MITO?

**Comment by MtthsLanger** [comment]
After running the "read_B1.R" script and excluding long-distance trips, the average trip lengths are the following:

"t.purpose"	"meanLength"
"HBE"	                          5.7
"HBO"	                          5.8
"HBS"	                          4.1
"HBW"	                        11.3
"NHBO"	                          6.1
"NHBW"	                          7.0

Joanna is informed.
My files with exact numbers are here: \\nas.ads.mwn.de\tubv\mob\indiv\matthias\mito

**Comment by moeckel** [comment]
Thank you, Matthias. I believe all numbers, except for the NHB values. When I worked with such data in the US, NHB trips were much shorter (around 2 miles). I would like to sit down with Matthias and review how NHB trips are defined in our MiD analysis. @joanna-ji: Please wait calibrating NHB trips until we had a chance to review the data.
